.github/workflows/test.yml

```diff
@@ -0,0 +1,16 @@
+on: [push,pull_request]
```

Comment:
these still don't specify the main branch, have a look at other examples

Reply:
Test.yml has been updated to specify the main branch